src/main/resources/application.properties

```diff
@@ -1,14 +1,14 @@
 # DATABASE CONFIGURATION
 spring.jpa.hibernate.ddl-auto=update
-#spring.jpa.database=mysql
-spring.jpa.database=postgresql
-#spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5Dialect
-#spring.datasource.url=jdbc:mysql://localhost:3306/task_crud?serverTimezone=Europe/Warsaw&useSSL=False
-spring.datasource.url=${JDBC_DATABASE_URL}
-#spring.datasource.username=kodilla
-spring.datasource.username=${JDBC_DATABASE_USERNAME}
-#spring.datasource.password=your_password
-spring.datasource.password=${JDBC_DATABASE_PASSWORD}
+spring.jpa.database=mysql
+#spring.jpa.database=postgresql
```

Comment:
zakomentowanego kodu nie trzymamy, od tego jest wersjonowanie :-)

Reply:
Wiem, ale w tym przypadku i w build.gradle jest to wskazanie z treści modułu.